content/docs/studio/get-started.md

```diff
@@ -75,25 +74,18 @@ details]).
    assignments.
 
    [models dashboard]:
-     /doc/studio/user-guide/model-registry/view-and-compare-models#models-dashboard
+     /doc/studio/user-guide/model-registry/view-models#models-dashboard
```

Comment:
Has this url changed in a different PR? In prod, it's still `view-and-compare..`